package.json

```diff
@@ -61,6 +60,7 @@
     "jwt-decode": "^2.2.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
+    "regenerator-runtime": "^0.12.1",
```

Comment:
Is that still necessary?

Reply:
It shouldn't be.

Reply:
I think it got there accidentally - will remove it.